.github/workflows/metrics.yml

```diff
@@ -0,0 +1,43 @@
+name: metrics
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    if: startsWith(github.repository, 'path-cc/')
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          persist-credentials: false  # use personal token, not GITHUB_TOKEN
```

Comment:
What's this for? Doesn't `GITHUB_TOKEN` have repo write access?

Reply:
I am not sure if we need it.  Do we have an example somewhere that does a git push to github?

I was starting with the example i found here:
https://github.com/marketplace/actions/github-push

Reply:
Dropping this in favor of doing a push similar to

https://github.com/path-cc/path-cc.github.io/blob/master/.github/workflows/web-preview.yml

since we're pushing to a separate repo now